imports/plugins/core/navigation/server/no-meteor/util/setDefaultsForNavigationTreeItems.js

```diff
@@ -1,10 +1,28 @@
+// Fallback options setting every value to false.
+// This effectively means the item will not be published to the public API.
+const standardDefaults = {
+  shouldNavigationTreeItemsBeAdminOnly: false,
+  shouldNavigationTreeItemsBeSecondaryNavOnly: false,
+  shouldNavigationTreeItemsBePublicallyVisible: false
+};
```

Comment:
Maybe you could just make `defaultValues` required and remove these standard fallbacks? Just seems like too many layers. There should always be default values returned from the server because you specified default values in registerPlugin.

Reply:
I'll make it required, add validation and throw an error if they're not defined.